CMakeLists.txt

```diff
@@ -17,14 +17,6 @@ if (WIN32)
 endif()
 add_subdirectory(third_party/pybind11)
 FIND_PACKAGE(Protobuf REQUIRED)
-include_directories(
```

Comment:
Why do we delete this?

Reply:
they are moved to individual libs (e.g. https://github.com/onnx/onnx/blob/a6f7955392ed7f4518c655edd17573ba01bf055a/CMakeLists.txt#L87)